lms/djangoapps/course_api/blocks/api.py

```diff
@@ -61,6 +64,16 @@ def get_blocks(
     # transform
     blocks = get_course_blocks(user, usage_key, transformers)
 
+    # filter blocks by types
+    if block_type_filter:
+        block_keys_to_remove = []
+        for block_key in blocks:
+            block_type = blocks.get_xblock_field(block_key, 'category')
+            if block_type not in block_type_filter:
```

Comment:
Hmm.. reading this line of code makes me 2nd guess the naming of the filter variable.  It's unclear from the name that it's a list.  Sorry for one more renaming, but how about changing it to: `block_types_filter` (with an "s")?